<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonOpenPredictorOutputFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Pred. Out Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMzFeaturesOutFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>621</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPredictorOutputFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>621</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenMzFeaturesOutFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Mz Feat. Out Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenOutFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Out Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOutFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>621</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelFilename">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditOut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>351</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditMzFeaturesFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>251</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditPredictorOutFiles">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>251</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditOutFileContent">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>251</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
